src/templates/below-component.js

```diff
@@ -0,0 +1,26 @@
+const { stylingTypes } = require('../helpers');
+
+const styled = `\
+const Wrapper = styled.div\`
+\`;
+`;
+
+const aphrodite = `\
+const styles = StyleSheet.create({
+  wrapper: {
+  },
```

Comment:
Nit: Same note as above; if possible with Prettier, can we add a blank line between open and close curlies?

Reply:
I think as Prettier introduces itself as an opinionated code formatter, it shouldn't be possible :) (Also I couldn't find an option to avoid it [here](https://github.com/prettier/prettier))

But maybe we can write a placeholder style for `wrapper` as create-react-app does that when you first init a project. I mean something like this:

```js
{
  wrapper: {
    height: 'auto',
    width: 'auto',
  }
}
}
```

What do you think? Happy to discuss!

Reply:
Its ok :) no change necessary, I'd rather not apply any styles, and it's super easy for a developer to just add that new line themselves when they want to add styles.